tooling/profiler/src/cli/execution_flamegraph_cmd.rs

```diff
@@ -66,15 +62,14 @@ fn run_with_generator(
     output_path: &Option<PathBuf>,
     pedantic_solving: bool,
     print_sample_count: bool,
-    verbose: bool,
 ) -> eyre::Result<()> {
     let program =
         read_program_from_file(artifact_path).context("Error reading program from file")?;
 
     ensure_brillig_entry_point(&program)?;
 
     if !print_sample_count && output_path.is_none() {
-        return report_error("Missing --output <OUTPUT> argument for when building a flamegraph")
+        return report_error("Missing --output <o> argument for when building a flamegraph")
```

Comment:
```suggestion
        return report_error("Missing --output <OUTPUT> argument for when building a flamegraph")
```

This change doesn't look correct.